webapi/src/main/java/org/sakaiproject/webapi/controllers/ProfileController.java

```diff
@@ -48,11 +53,22 @@ public class ProfileController extends AbstractSakaiApiController {
     @Autowired private ProfileLogic profileLogic;
     @Autowired private UserDirectoryService userDirectoryService;
     @Autowired private ServerConfigurationService serverConfigurationService;
+    @Autowired private SecurityService securityService;
+    @Autowired private SiteService siteService;
+    @Autowired private ToolManager toolManager;
+
+    // Permission constants for Roster tool
+    private static final String ROSTER_PERMISSION_VIEW_PROFILE = "roster.viewprofile";
+    private static final String ROSTER_PERMISSION_VIEW_EMAIL = "roster.viewemail";
+    private static final String ROSTER_PERMISSION_VIEW_CANDIDATE_DETAILS = "roster.viewcandidatedetails";
 
```

Comment:
I think you can use the entries from RosterFunctions instead, better for maintenance

Reply:
yeah giving it a try ....

Reply:
thanks. done. permissions now refer to roster-api

Reply:
i can't figure this out .... do you have any idea?

Reply:
roster2 does not export an API